spec/models/manageiq/providers/ansible_tower/configuration_manager/refresh_parser_spec.rb

```diff
@@ -15,6 +15,7 @@
   let(:all_job_templates) { (1..2).collect { |i| AnsibleTowerClient::JobTemplate.new("id" => i, "name" => "template#{i}", "description" => "description#{i}", "extra_vars" => "some_json_payload") } }
 
   it "#configuration_manager_inv_to_hashes" do
+    allow_any_instance_of(AnsibleTowerClient::JobTemplate).to receive(:survey_spec).and_return('some_hash_payload')
```

Comment:
@syncrou We don't usually do this.  Normally we ensure that the environment has a jobtemplate with a survey spec then look for that instance in the database and verify that the correct data was refreshed.